### DISCUSSION TITLE
HuBMAP: Exploration User Interface (EUI)

### DISCUSSION THREAD

**Discussion by andreasbueckle** [nih-cfde/training-and-engagement#417] [Common Fund Tools]
In this board, please ask your questions and/or suggestions for improving the functionality of the EUI so we can then forward them to our developer team.

Link to the EUI: https://portal.hubmapconsortium.org/ccf-eui

**Comment by s-canchi** [comment]
What is the data confidential policy when exploring our own data in these RUI/EUI?

**Reply by andreasbueckle** [reply]
Data confidentiality guidelines are controlled by the data provider. Each non-HuBMAP data provider controls access to their own data by deciding what data is linked from the RUI portal. Many consortia have linked registered sample locations to publications, in order to prevent unintended access to data that has restrictions on how it is shared.

**Comment by s-canchi** [comment]
The tissue blocks in the paper linked with SPARC- are those basically tissue blocks they assayed? Even with the kidney earlier and those "blocks", are they the data on tissue blocks in a human that they have assayed and then integrated into hubmap?

**Reply by emquardokus** [reply]
Tissue blocks registered from other consortia like SPARC, and KPMP will allow the data from them to be accessed, compared and queried from within HuBMAP data portal.
